packages/agent/src/agent-main.test.ts

```diff
@@ -150,6 +151,7 @@ describe('Main', () => {
     fetchSpy.mockImplementation(async () => {
       return {
         ok: true,
+        headers: new Headers(),
```

Comment:
Interesting.  I assume this is because any mock request will throw if `headers` is `undefined`?  

Any concerns about this impacting other customers in their unit tests?  I guess it will only happen at dev-time, and this is technically correct / technically required?